### ISSUE TITLE
SCSSLint::Linter::Indentation 'undefined method `[]' for nil:NilClass'

### ISSUE THREAD

**Issue by lencioni** [brigade/scss-lint#700]
scss lint seems to choke on a file that uses windows-style newlines with carriage returns. To type a windows-style newline with carriage return in Vim, press `<Ctrl>+v<Ctrl>+m` in insert mode. This will show up as `^M`.

```
SCSSLint::Linter::Indentation raised unexpected error linting file ./app/assets/stylesheets/packages/wishlists.scss: 'undefined method `[]' for nil:NilClass'
/Users/joe_lencioni/.rbenv/versions/2.0.0-p481/lib/ruby/gems/2.0.0/gems/scss_lint-0.44.0/lib/scss_lint/linter/indentation.rb:194:in `node_indent'
/Users/joe_lencioni/.rbenv/versions/2.0.0-p481/lib/ruby/gems/2.0.0/gems/scss_lint-0.44.0/lib/scss_lint/linter/indentation.rb:44:in `check_indent_width'
/Users/joe_lencioni/.rbenv/versions/2.0.0-p481/lib/ruby/gems/2.0.0/gems/scss_lint-0.44.0/lib/scss_lint/linter/indentation.rb:40:in `check_indentation'
/Users/joe_lencioni/.rbenv/versions/2.0.0-p481/lib/ruby/gems/2.0.0/gems/scss_lint-0.44.0/lib/scss_lint/linter/indentation.rb:102:in `visit_import'
/Users/joe_lencioni/.rbenv/versions/2.0.0-p481/lib/ruby/gems/2.0.0/gems/sass-3.4.21/lib/sass/tree/visitors/base.rb:36:in `visit'
/Users/joe_lencioni/.rbenv/versions/2.0.0-p481/lib/ruby/gems/2.0.0/gems/scss_lint-0.44.0/lib/scss_lint/linter.rb:138:in `visit'
/Users/joe_lencioni/.rbenv/versions/2.0.0-p481/lib/ruby/gems/2.0.0/gems/scss_lint-0.44.0/lib/scss_lint/linter.rb:149:in `block in visit_children'
/Users/joe_lencioni/.rbenv/versions/2.0.0-p481/lib/ruby/gems/2.0.0/gems/scss_lint-0.44.0/lib/scss_lint/linter.rb:147:in `each'
/Users/joe_lencioni/.rbenv/versions/2.0.0-p481/lib/ruby/gems/2.0.0/gems/scss_lint-0.44.0/lib/scss_lint/linter.rb:147:in `visit_children'
/Users/joe_lencioni/.rbenv/versions/2.0.0-p481/lib/ruby/gems/2.0.0/gems/sass-3.4.21/lib/sass/tree/visitors/base.rb:36:in `block in visit'
/Users/joe_lencioni/.rbenv/versions/2.0.0-p481/lib/ruby/gems/2.0.0/gems/scss_lint-0.44.0/lib/scss_lint/linter/indentation.rb:19:in `visit_root'
/Users/joe_lencioni/.rbenv/versions/2.0.0-p481/lib/ruby/gems/2.0.0/gems/sass-3.4.21/lib/sass/tree/visitors/base.rb:36:in `visit'
/Users/joe_lencioni/.rbenv/versions/2.0.0-p481/lib/ruby/gems/2.0.0/gems/scss_lint-0.44.0/lib/scss_lint/linter.rb:138:in `visit'
/Users/joe_lencioni/.rbenv/versions/2.0.0-p481/lib/ruby/gems/2.0.0/gems/scss_lint-0.44.0/lib/scss_lint/linter.rb:41:in `run'
/Users/joe_lencioni/.rbenv/versions/2.0.0-p481/lib/ruby/gems/2.0.0/gems/scss_lint-0.44.0/lib/scss_lint/runner.rb:51:in `run_linter'
/Users/joe_lencioni/.rbenv/versions/2.0.0-p481/lib/ruby/gems/2.0.0/gems/scss_lint-0.44.0/lib/scss_lint/runner.rb:33:in `block in find_lints'
/Users/joe_lencioni/.rbenv/versions/2.0.0-p481/lib/ruby/gems/2.0.0/gems/scss_lint-0.44.0/lib/scss_lint/runner.rb:31:in `each'
/Users/joe_lencioni/.rbenv/versions/2.0.0-p481/lib/ruby/gems/2.0.0/gems/scss_lint-0.44.0/lib/scss_lint/runner.rb:31:in `find_lints'
/Users/joe_lencioni/.rbenv/versions/2.0.0-p481/lib/ruby/gems/2.0.0/gems/scss_lint-0.44.0/lib/scss_lint/runner.rb:19:in `block in run'
/Users/joe_lencioni/.rbenv/versions/2.0.0-p481/lib/ruby/gems/2.0.0/gems/scss_lint-0.44.0/lib/scss_lint/runner.rb:18:in `each'
/Users/joe_lencioni/.rbenv/versions/2.0.0-p481/lib/ruby/gems/2.0.0/gems/scss_lint-0.44.0/lib/scss_lint/runner.rb:18:in `run'
/Users/joe_lencioni/.rbenv/versions/2.0.0-p481/lib/ruby/gems/2.0.0/gems/scss_lint-0.44.0/lib/scss_lint/cli.rb:73:in `scan_for_lints'
/Users/joe_lencioni/.rbenv/versions/2.0.0-p481/lib/ruby/gems/2.0.0/gems/scss_lint-0.44.0/lib/scss_lint/cli.rb:58:in `act_on_options'
/Users/joe_lencioni/.rbenv/versions/2.0.0-p481/lib/ruby/gems/2.0.0/gems/scss_lint-0.44.0/lib/scss_lint/cli.rb:32:in `run'
/Users/joe_lencioni/.rbenv/versions/2.0.0-p481/lib/ruby/gems/2.0.0/gems/scss_lint-0.44.0/bin/scss-lint:7:in `<top (required)>'
/Users/joe_lencioni/.rbenv/versions/2.0.0-p481/bin/scss-lint:23:in `load'
/Users/joe_lencioni/.rbenv/versions/2.0.0-p481/bin/scss-lint:23:in `<main>'
Report this bug at https://github.com/brigade/scss-lint/issues
```
